message.txt

```diff
@@ -1,4 +1,6 @@
-Hello, world... I am a message. Thank you for reading.
+where your Thoughts End, (?)
 
-peace suckas
- end
+and reality beGins (!)
+
+
+is a very Thin LINE... end
```

Comment:
weirdness aside, this message tests punctuation/special characters, line breaks, and letter case